milli/src/update/index_documents/extract/mod.rs

```diff
@@ -55,7 +55,7 @@ pub(crate) fn data_from_obkv_documents(
         .collect::<Result<()>>()?;
 
     #[allow(clippy::type_complexity)]
-    let result: Result<(Vec<_>, (Vec<_>, (Vec<_>, Vec<_>)))> = flattened_obkv_chunks
+    let result: Result<(Vec<_>, (Vec<_>, (Vec<_>, (Vec<_>, Vec<_>))))> = flattened_obkv_chunks
```

Comment:
tbh I'm a bit with clippy on this one 😅 

I'm not sure what we could do though to express the type differently (using a type alias wouldn't make me feel better and would prevent inference from doing its magic)

Reply:
What? Don’t you like lisp?

Reply:
I will keep this type for now as I am not sure how to fix that, it is related to the collect and `FromIterator` standard implementation trait.